core/src/main/scala/ai/h2o/sparkling/backend/utils/SharedBackendUtils.scala

```diff
@@ -185,14 +179,21 @@ trait SharedBackendUtils extends Logging with Serializable {
       .add(getH2OSecurityArgs(conf))
       .addIf("-quiet", !conf.clientVerboseOutput)
       .add("-log_level", conf.h2oClientLogLevel)
-      .add("-log_dir", conf.h2oClientLogDir)
       .add("-baseport", conf.clientBasePort)
       .add("-flow_dir", conf.flowDir)
       .add("-port", Some(conf.clientWebPort).filter(_ > 0))
       .addAsString(conf.clientExtraProperties)
       .buildArgs()
   }
 
+  private def determineLogDir(conf: H2OConf): String = {
+    val fs = java.io.File.separator
```

Comment:
nit: WDYT about using `Paths.get `instead of using file separator multiple times

Reply:
👍 great idea